pkg/tracing/setup.go

```diff
@@ -62,6 +62,16 @@ var (
 	}
 )
 
+var (
+	// OnePercentSampling is a configuration that samples 1% of the requests.
+	OnePercentSampling = &tracingconfig.Config{
+		Enable:         true,
+		Debug:          false,
+		SampleRate:     0.01,
+		ZipkinEndpoint: "http://zipkin.istio-system.svc.cluster.local:9411/api/v2/spans",
```

Comment:
Should ZipkinEndpoint be configurable? I can imaging admins wanting to run zipkin outside the prod cluster.

Reply:
It absolutely should be. I verified that even if this endpoint doesn't exist, the Sources still work correctly. So even with an incorrect endpoint, users and admins are no worse off than today.

I think the answer in the long term is to use something similar to the rest of eventing, read the `config-tracing` ConfigMap. However, I see problems with the two ways we can do this:
* Watch the ConfigMaps - This is what all the other components do. This is OK because we control the ServiceAccount they use and ensure it has watch permissions on CofigMaps. These sources run as arbitrary ServiceAccounts, so we can't guarantee they have permission. I have not investigated whether our code handles lack of permission as just don't read the ConfigMap or stop everything from working.
* Mount the ConfigMap as a volume - We aren't sure the ConfigMap will exist. Users will create it only if they want it. Even if the reconciler checks the existence of the ConfigMap, then either adds the volume mount or not, we would need to bounce the Source Pods if the ConfigMap was created/deleted.

So because I don't have a good answer for those, I am just putting in the ground work. Tracing works, but with a hard coded config. Either I when I have more time, or someone else who is interested, can figure out how to get dynamic configuration in here.